integration/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/MicroRegistryManager.java

```diff
@@ -123,15 +123,25 @@ public void updateResource(String resourcePath, String sourcePath, boolean backu
     /**
      * Function is to check if the registry resources are exist in filepath
      *
-     * @param sourcePath path to registry resource
      * @param resourcePath path to resource file path from registry resource
      * @param filename registry filename
      */
 
-    public boolean checkResourceExist(String sourcePath, String resourcePath, String filename) throws Exception {
-        try {
-            String path = sourcePath + resourcePath + filename;
-            return path.contains(filename);
+    public boolean checkResourceExist(String resourcePath, String filename) throws Exception {
+
+        StringBuilder pathBuilder = new StringBuilder();
+        pathBuilder.append(System.getProperty("carbon.home")).append(File.separator).append("registry").append(File.separator);
+
+        String relativePath = resourcePath.substring(5).replace('/', File.separatorChar);
+        pathBuilder.append("config").append(relativePath);
+
+        boolean isExists = new File(pathBuilder + filename).isFile();
+        try{
+            if (isExists){
```

Comment:
We can directly return new File(pathBuilder + filename).isFile()

Reply:
There are formatting issues and no need to have try-catch block for this condition check

Reply:
Added the changes. Please review.